protos/telemetry/telemetry.proto

```diff
@@ -569,6 +585,14 @@ message Odometry {
     Covariance velocity_covariance = 9; // Velocity cross-covariance matrix.
 }
 
+// DistanceSensor message type.
+message DistanceSensor {
+
+    uint64 min_distance=1; // Minimum distance the sensor can measure.
```

Comment:
What's the unit? And what about using `float`?

Reply:
And do we need multiple instances?

Reply:
This is my suggestion:
```suggestion
    float min_distance_m = 1; // Minimum distance the sensor can measure.
```

And then the same for the others.

Reply:
And you could make it a repeated field, one entry per `id` in https://mavlink.io/en/messages/common.html#DISTANCE_SENSOR.